contracts/token/ERC721/Tatum721.sol

```diff
@@ -8,7 +8,8 @@ import "../../access/AccessControlEnumerable.sol";
 contract Tatum721 is ERC721Enumerable, ERC721URIStorage, AccessControlEnumerable {
 
     bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
-
+    mapping (uint256 => address) private _cashbacks;
+    mapping (address => uint256) private _cashbacksValue;
```

Comment:
this should be mapping (uint256 => uint256[]). 1 token can have multiple cashback values. Cashback value is basically a threshold and defines how much should user pay in ETH or CELO during the transfer of the token. It's not a total amount of cashback paid.